functions/index.js

```diff
@@ -0,0 +1,8 @@
+const functions = require('firebase-functions');
+
+// // Create and Deploy Your First Cloud Functions
```

Comment:
Do you still need this? Is this auto-generated?

Reply:
Hmm... not sure about that - if we're still using it. I'll look into it.

Reply:
It looks like this [Get Started](https://firebase.google.com/docs/functions/get-started) firebase doc mentions this. It looks like we would only need it if we wanted to build or connect cloud services to the project, if I'm understanding that correctly. Does that sound right? If that's the case then we could probably get rid of it, or we could leave it in case we decide to get fancy in the future... 😄 What do you think @laurenmbeatty ?